src/ai-bolit.php

```diff
@@ -2972,7 +3009,7 @@ function CriticalPHP($l_FN, $l_Index, $l_Content, &$l_Pos, &$l_SigId)
 
 if (AI_EXPERT) {
   foreach ($gX_DBShe as $l_Item) {
-    $l_Pos = stripos($l_Content, $l_Item);
+    $l_Pos = strpos($l_Content_lo, $l_Item);

```

Comment:
Why did you add AI_EXPERT check for the $gX_DBShe ? It has to scan even when AI_EXPERT is 0.